src/components/loggedOut/loggedOutNavBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
Use a prettier extension to get rid of the random white space, the code looks very funky

Reply:
I've added the prettier extension and formatted the files